Lesson_5/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+    font-family: 'Jaldi';
+    margin: 0;
+    background-color: rgb(162, 214, 212);
+
+}
+.buttonBlock {
+    align-items: center;
+    margin-left: 45%;
+    margin-top: 10%;
+}
+.newclass {
+    padding: 20px;
+    border: 15px;
+    background-color: grey;
+    color: black;
+    text-align: center;
+    font-size: 20px;
+}
+.big-button {
```

Comment:
This way of naming css classes is also valid, so choose one for your css file and use only one format for all classes.